index.html

```diff
@@ -454,58 +463,47 @@ <h4>Script Type</h4>
+                or to ensure that the words can fit within the time available in an audio description workflow.</p>
+                <p><a>Script Events</a> in this type of <a>script</a> SHOULD contain <a>Text</a> objects
+                  in the <a>Target Recording Language</a>.
+                  They MAY also contain <a>Original</a> <a>Text</a> objects from the <a>Original Language Transcript</a>
+                  in the case that the <a>Original</a> language is not the <a>Target Recording Language</a>, 
+                  for context, to assist further processing.</p>
+
                 <aside class="note">The <a>Script Type</a> of a <a>DAPT Script</a> cannot necessarily be detected by inspecting the text content of the document.
-                For example, the adaptation of a <a>Translated Transcript</a> into a <a>Pre-recording Dub Script</a>
+                For example, the adaptation of a <a>Translated Transcript</a> into a <a>Pre-recording Script</a>
                 can consist in replacing some words in the text content of a <a>Script Event</a> without altering the rest of the document.
                 In either case, <a>Text</a> objects that are translations have <a>Text Language Source</a> properties set to <a>Translation</a>.</aside>
-              </li>
-              <li>
-                <dfn>As-recorded Dub Script</dfn>:
-                <p>When the <code>daptm:scriptType</code> value is <code>DUBBING_AS_RECORDED</code>,
-                  the document represents a transcription of the output of the dubbing workflow.</p>
-                <p><a>Script Events</a> in this type of <a>script</a> SHOULD contain <a>Translation</a> <a>Text</a> objects
-                  in the <a>Target Recording Language</a>
-                  and MAY also contain <a>Original</a> <a>Text</a> objects from the <a>Original Language Transcript</a>
-                  or <a>Translation</a> <a>Text</a> objects in other languages for for context and quality verification.</p>
-                  <aside class="note">Even though the output of the dubbing workflow is in the same language as those
-                  <a>Text</a> objects with the matching language,
-                  the <a>Text Language Source</a> of those Text objects remains <a>Translation</a>
-                  since they differ from the original language of the dialogue.</aside>
-              </li>
-              <li>
-                <dfn>Audio Description Script</dfn>:
-                <p>When the <code>daptm:scriptType</code> value is <code>AUDIO_DESCRIPTION</code>,
-                  the document represents a <a>transcript</a> of the important visual contents of the
-                  video written to be suitable for use as a <a>script</a>
-                  for generating an audio representation.
-                  It contains text, times, and optional links to audio and mixing instructions
-                  for the purpose of producing an audio description audio track.</p>
```

Comment:
I just noticed we deleted this useful sentence by accident. Suggestion added for putting an equivalent back in.